assets/src/components/ProgressBarV2.js

```diff
@@ -37,6 +37,12 @@ function ProgressBarV2 (props) {
     return value / maxValue * 100
```

Comment:
Optional: Also, remove the unused import on L6

Reply:
done